src/components/__tests__/LibrariesPage-test.tsx

```diff
@@ -100,18 +99,12 @@ describe("LibrariesPage", () => {
     expect(searchForm.find(".btn").length).to.equal(2);
   });
 
-  it("should search", async() => {
-    let form = wrapper.find(Form).at(0);
-    const libraryPageSearch = () => {
-      wrapper.setState({ showAll: false });
-      search();
-    };
-    let formSubmit = Sinon.stub(form.instance(), "submit").callsFake(libraryPageSearch);
-    let spyClear = Sinon.spy(wrapper.instance(), "clear");
-
```

Comment:
It seemed this didn't do anything. I think the search method and functionality could be better tested.